wts.go

```diff
@@ -60,7 +60,7 @@ type WTSClientInfo struct {
 	InitialProgram      string
 	EncryptionLevel     byte
 	ClientAddressFamily AddressFamily
-	clientAddress       [wrappers.CLIENTADDRESS_LENGTH + 1]uint16
+	ClientAddress       [wrappers.CLIENTADDRESS_LENGTH + 1]uint16
```

Comment:
Any reason for changing this?  We had decided in the previous PR to keep it private for now because we weren't sure of the right representation to expose.

Reply:
This makes it easier to debug WTS. For example I can print all WTSClientInfo fields and check what is in clientAddress. Without this is impossible because ClientAddressToIP supports only IPv4 (and now IPv6)

Reply:
If it's just for debugging, then how about temporarily changing it when you need to print it?  I'd rather avoid exposing an API that we might want to change in the future.

Reply:
Honestly, I have additional programs, eg for displaying information about the WTS. We use them for debugging problems, but remembering to change clientAddress to ClientAddress when compiling will be a hassle. Additionally: now we do not support all possible types of addresses that may appear in this field. If we make this private field, someone using the library in the case of a different type of address will not be able to read it.

Reply:
Sure, but the solution to that is to add support for other address types to the library, not to expose internals to end users.

Reply:
Right. But I have no idea what is the data format in ClientAddress for other values. So what do you prefer: leave it public or private?

Reply:
Let's stick with private for now.